server/athenian/api/align/directives/date.py

```diff
@@ -0,0 +1,65 @@
+from typing import Any, Union
+
+from ariadne import SchemaDirectiveVisitor
+from graphql import default_field_resolver
+from graphql.type import get_named_type, GraphQLField, GraphQLInputField, GraphQLInputObjectType, \
+    GraphQLInterfaceType, GraphQLNonNull, GraphQLObjectType, GraphQLScalarType
+
+from athenian.api.serialization import deserialize_date, serialize_date
+
+
+class DateDirective(SchemaDirectiveVisitor):
+    """GraphQL directive for a string field expressing a date."""
+
+    def visit_field_definition(
+        self,
+        field: GraphQLField,
+        object_type: Union[GraphQLObjectType, GraphQLInterfaceType],
+    ) -> GraphQLField:
+        """Wrap field resolver to convert resolved date value to a string."""
+        original_resolver = field.resolve or default_field_resolver
+
+        def resolve_formatted_date(obj, info, **kwargs):
+            result = original_resolver(obj, info, **kwargs)
+            if result is None:
+                return None
+            return result.isoformat()
```

Comment:
Is this used? Better `serialize_datetime`

Reply:
done